### DISCUSSION TITLE
Consider tweek to dashtop

### DISCUSSION THREAD

**Discussion by jepingel293874** [tanelpoder/tpt-oracle#32] [Ideas]
For reasons less than obvious to me, the inst_id column in GV$ACTIVE_SESSION_HISTORY was changed to instance_number in DBA_HIST_ACTIVE_SESS_HISTORY.   I understand you had no input in this odd choice, but in pursuit of quickly switching from ashtop to dashtop where inst_id is used as a projection or filter, would it be possible for you to change dashtop code?

        FROM dba_hist_active_sess_history a) a

To
        FROM (select instance_number inst_id, x.* from dba_hist_active_sess_history x) a) a

**Comment by tanelpoder** [comment]
Good idea, this would make life a little easier. This inline view will probably be merged with the parent query, so wouldn't likely cause any performance/complexity issues. Would you be able to make this change and test in your environment and then issue a pull request?

**Reply by ds83x** [reply]
Hello!
May I proceed with changing the dashtop.sql to have the inst_id column alias?

**Comment by tanelpoder** [comment]
Yes please - and once you're done & tested it, send a PR.

Thanks!
